### ISSUE TITLE
Failing rubocop's cop for grape gem

### ISSUE THREAD

**Issue by gogainda** [oracle/truffleruby#2084]
For some reason when I ran the grape test suit it fails with 
```
An error occurred while Style/FormatStringToken cop was inspecting /home/travis/build/ruby-grape/grape/lib/grape/exceptions/validation_errors.rb:9:30.
```
but it doesn't for Ruby MRI

The tests are passing btw, but this issue is blocking me from enabling Truffleruby head on grape's CI

Full logs are [here](https://travis-ci.org/github/ruby-grape/grape/jobs/722691736)

**Comment by jaime-rave-conichi** [comment]
That might be related to https://github.com/oracle/truffleruby/issues/1913

**Comment by eregon** [comment]
I believe this is fixed in [RuboCop 0.90](https://github.com/rubocop-hq/rubocop/releases/tag/v0.90.0) and is a duplicate of https://github.com/oracle/truffleruby/issues/1913.
@gogainda Could you verify it works fine with RuboCop 0.90?

**Comment by gogainda** [comment]
@eregon @jaime-rave-conichi yep, rubocop works fine now
